internal/credential/vault/repository_credentials.go

```diff
@@ -38,6 +41,16 @@ func (r *Repository) Issue(ctx context.Context, sessionId string, requests []cre
 		if err != nil {
 			return nil, err
 		}
+
+		period, err := parseutil.ParseDurationSecond(cred.getExpiration())
+		if err != nil {
+			return nil, fmt.Errorf("unable to parse vault token period")
+		}
+		runJobsInterval := r.scheduler.GetRunJobsInterval()
+		if period < runJobsInterval {
+			event.WriteError(ctx, op, fmt.Errorf("WARNING: token ttl expires before job scheduler can run"), event.WithInfo("runJobsInterval", runJobsInterval))
```

Comment:
This is calling out a warning on the token ttl but should probably be `WARNING: credential expires before...`